config/finance.spc

```diff
@@ -1,3 +1,8 @@
 connection "finance" {
   plugin = "finance"
+
+  # IEX Cloud authenticates your API requests using your account’s API tokens.
```

Comment:
We should mention this token is only used for the `finance_us_sec_public_company` table and that a publishable token is OK